module/plugins/hoster/MegaDebridEu.py

```diff
@@ -28,16 +30,19 @@ class MegaDebridEu(MultiHoster):
 
     API_URL = "https://www.mega-debrid.eu/api.php"
 
+    def api_response(self, get={}, post={}):
+        self.req.http.c.setopt(pycurl.USERAGENT, encode('Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36'))
```

Comment:
Default user-agent doesn't return json.
I have not found how to change this otherwise.